build/wget/patches/autoconf.patch

```diff
@@ -0,0 +1,18 @@
+--- a/m4/std-gnu11.m4~	2020-12-22 21:07:39.141547631 +0000
++++ a/m4/std-gnu11.m4	2020-12-22 21:07:41.189875210 +0000
+@@ -86,12 +86,9 @@
+ dnl Set ac_prog_cc_stdc to the supported C version.
+ dnl Also set the documented variable ac_cv_prog_cc_stdc;
+ dnl its name was chosen when it was cached, but it is no longer cached.
+-_AC_PROG_CC_C11([ac_prog_cc_stdc=c11
+-		 ac_cv_prog_cc_stdc=$ac_cv_prog_cc_c11],
+-  [_AC_PROG_CC_C99([ac_prog_cc_stdc=c99
+-		    ac_cv_prog_cc_stdc=$ac_cv_prog_cc_c99],
+-     [_AC_PROG_CC_C89([ac_prog_cc_stdc=c89
+-		       ac_cv_prog_cc_stdc=$ac_cv_prog_cc_c89],
++_AC_PROG_CC_C11([ac_prog_cc_stdc=c11 ac_cv_prog_cc_stdc=$ac_cv_prog_cc_c11],
++  [_AC_PROG_CC_C99([ac_prog_cc_stdc=c99 ac_cv_prog_cc_stdc=$ac_cv_prog_cc_c99],
++     [_AC_PROG_CC_C89([ac_prog_cc_stdc=c89 ac_cv_prog_cc_stdc=$ac_cv_prog_cc_c89],
```

Comment:
that seems only a formatting change ...

Reply:
Yes - the patch file could do with a comment, I've added one.